src/Pagination.vue

```diff
@@ -29,35 +36,66 @@ export default {
   name: "pagination",
   data() {
     return {
+      /**
+       * link on Carousel
+       */
       parentContainer: this.$parent
     };
   },
   methods: {
+    /**
+     * Change page by index
+     * @param {number} index
+     * return {void}
+     */
     goToPage(index) {
+      /**
+       * @event paginationclick
+       * @type {number}
+       */
       this.$emit("paginationclick", index);
+    },
+
+    /**
+     * Check on current dot
+     * @param {number} index - dot index
+     * @return {boolean}
+     */
+    isCurrentDot(index) {
```

Comment:
Good thinking here

Reply:
I always prefer use methods and computed properties for styling html elements. I want here too use this, but i think it's will be overmuch for me (too blatantly).